codemeta.json

```diff
@@ -5,34 +5,36 @@
     "codeRepository": "https://github.com/javierrico/gLike",
     "dateCreated": "2018-11-13",
     "datePublished": "2020-07-30",
```

Comment:
Same question, what is this date? not today?

Reply:
This should be their meaning:
- `"dateCreated"` -> date in which the software was created (i.e. made public on GitHub);
- `"datePublsihed"` -> date of the first release (on GitHub);
- `"dateModified"` -> date of the last modification of this entry, i.e. the date in which the lates release is made.

Unfortunately the tool provided by the OSSR does not convert `"dateModified"` of the codemeta into "publication date" - of the zenodo as it should be. So it has to be corrected manually. Which is what I just did.